resources/chromium/generic_sensor_mocks.js

```diff
@@ -201,10 +201,10 @@ var GenericSensorTest = (() => {
       if (testInternal.initialized)
         throw new Error('Call reset() before initialize().');
 
-      if (testRunner) { // Grant sensor permissions for Chromium testrunner.
+      if (window.testRunner) { // Grant sensor permissions for Chromium testrunner.
```

Comment:
Sorry if this sounds dumb, but isn't `testRunner` the same as `window.testRunner` here? Or do the Mojo JS files behave differently?

Reply:
This `testRunner` is specific for Chromium testrunner, not sure if it is still useful in `LayoutTests/external/wpt`, but for sure, it is `undefined` in w-p-t runnner.

Reply:
i.e. tests will fail with `Uncaught ReferenceError: testRunner is not defined`